python/lsst/dax/distribution/testingdistrib.py

```diff
@@ -184,9 +186,9 @@ def run(self):
             while len(completedChunks) > 0:
                 completedChunks = client.clientReportChunksComplete(completedChunks)
 
-
         print("ClientTestThrd.run finished")
-        if self.success is None: self.success = True
+        if self.success is None:
+            self.success = True
```

Comment:
Same as above, print message to track this special case of success?

Reply:
This one actually serves a purpose. Since success = None until the job is complete, other entities can test success and see that the job isn't done, and if an exception is raised, this line is never called and success remains not True.